Add simple peak detection

ChristophWurst opened the pull request:
Fixes https://github.com/nextcloud/suspicious_login/issues/115

rullzer commented on lib/Service/LoginClassifier.php:
```diff
@@ -148,13 +150,31 @@ private function persistSuspiciousLogin(string $uid, string $ip) {
+			return;
+		}
+
+		// Have we sent more than 10 alerts in the last week
```
```suggestion
		// Have we sent more than 20 alerts in the last week
```

rullzer replied:
Also this is the last 2 days.... not last week...

rullzer commented on lib/Service/LoginClassifier.php:
```diff
@@ -148,13 +150,31 @@ private function persistSuspiciousLogin(string $uid, string $ip) {
+		// Have we sent more than 10 alerts in the last week
+		$lastTwoDays = count($this->mapper->findRecentByUid($uid, $now - 60 * 60 * 24 * 2));
+		if ($lastTwoDays >= 20) {
+			$this->logger->warning("Suspicious login peak detected: $uid received $lastTwoDays alerts in the last two hours");
```
Also doesn't seem correct

rullzer commented on lib/Service/LoginClassifier.php:
```diff
@@ -148,13 +150,31 @@ private function persistSuspiciousLogin(string $uid, string $ip) {
+		// Have we sent more than three alerts in the last hour
+		$lastHour = count($this->mapper->findRecentByUid($uid, $now - 60 * 60));
+		if ($lastHour >= 3) {
+			$this->logger->warning("Suspicious login peak detected: $uid received $lastTwoDays alerts in the last hour");
```
`$lastTwoDays` seems incorrect

rullzer replied:
```suggestion
			$this->logger->warning("Suspicious login peak detected: $uid received $lastHour alerts in the last hour");
```

ChristophWurst commented:
Please have another look.

* Removed the comments as they are prone to get out of sync with the code/notification
* Changed the comparison to `>` as opposed to `>=` as the current alert is included
* Reduced to 10 alerts per two days window max
